README.md

```diff
@@ -0,0 +1,155 @@
+# TAD_Pathways
+
+## Leveraging TADs to identify candidate genes at GWAS signals
+
+**Gregory P. Way and Casey S. Greene - 2017**
+
+### Summary
+
+The repository contains data and instructions to implement a "TAD_Pathways"
+analysis for over 300 different trait/disease GWAS or custom SNP lists.
+
+TAD_Pathways uses the principles of topologically association domains (TADs) to
+define where an association signal (typically a GWAS signal) can most likely
+impact gene function. We use TAD boundaries as defined by
+[Dixon et al. 2012](https://doi.org/10.1038/nature11082) and
+[hg19 Gencode genes](ftp://ftp.sanger.ac.uk/pub/gencode/Gencode_human/release_19/)
+to identify which genes may be implicated. We then input this list into a
+[WebGestalt Pathways Analysis](http://webgestalt.org/) to output
+significantly associated pathways implicated by the input TAD-defined geneset.
+
+For more specific details about the method, refer to our
+[preprint](https://doi.org/10.1101/087718).
```

Comment:
Not essential but is nice to set title like:

```md
[preprint](https://doi.org/10.1101/087718 "Determining causal genes from GWAS signals using topologically associating domains")
```

This will be convenient for readers and may help with SEO.